x-pack/plugins/upgrade_assistant/server/routes/cluster_checkup.test.ts

```diff
@@ -62,11 +52,11 @@ describe('cluster checkup API', () => {
         nodes: [],
       });
 
-      await serverShim.router.getHandler({
+      await routeDependencies.router.getHandler({
         method: 'get',
         pathPattern: '/api/upgrade_assistant/status',
-      })(ctxMock, createRequestMock(), kibanaResponseFactory);
-      expect(spy.mock.calls[0][2]).toBe(true);
+      })(routeHandlerContextMock, createRequestMock(), kibanaResponseFactory);
+      expect(spy.mock.calls[0][1]).toBe(true);
```

Comment:
nit: This kind of tests are very obscure, but I know that this PR is not to improve them!